### DISCUSSION TITLE
Which rule would throw on same property destructured multiple times from the object?

### DISCUSSION THREAD

**Discussion by niksumeiko** [eslint/eslint#17801] [Q&A]
```ts
const {
    errors: formikErrors,
    errors: {
        receiverName: receiverNameError,
        receiverAddress: receiverAddressError,
    },
    errors: anotherErrors,
} = useFormikContext();
```

Eslint doesn't throw on the same `errors` property destructured multiple times.

What rule would make it fail? And does it make sense, in your opinion?

**Comment by Rec0iL99** [comment]
Hi @niksumeiko, I don't think there is a rule for catching this currently in the core.

[rules reference](https://eslint.org/docs/latest/rules/)
